Solution of WebShop/WebShop.Tricentis.Framework/PageObject/AuthorizationPage.cs

```diff
@@ -8,5 +9,9 @@ namespace WebShop.Tricentis.Framework.PageObject
 {
     class AuthorizationPage
     {
+        private readonly By _inputEmail = By.CssSelector("[name='Email']");
+        private readonly By _inputPassword = By.CssSelector("[name='Password']");
+        private readonly By _chekboxRememberMe = By.CssSelector("#RememberMe");
+        private readonly By _loginButton = By.CssSelector("[class='button-1 login-button']");
```

Comment:
Please, change [class=button-1 to .button-1
also, please try find more beauty selector

Reply:
Changed